src/components/Pagination/types.ts

```diff
@@ -0,0 +1,8 @@
+export type PaginationProps = {
+   info: {
+      total: number;
+      page: number;
+      limit: number;
+   };
+   onChange?: Function;
```

Comment:
```suggestion
   onChange?: (page: number) => void;
```
Nên định nghĩa type rõ ràng hơn